UACloudLibraryServer/appsettings.json

```diff
@@ -6,5 +6,18 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "DataPlaneSdk": {
+    "ControlApi": {
+      "BaseUrl": "http://localhost:8083/api/control"

```

Comment:
avoid hard-coded dummy values in production config. better to leave out those values (or leave them blank) and provide an additional `appsettings.demo.json` or something and overwrite the values there.

Reply:
Fixed.